build.sh

```diff
@@ -61,6 +61,9 @@ case "$TRAVIS_OS_NAME" in
     echo 'Install npm packages for Linux'
     npm install --save-dev electron-installer-debian --silent
 
+    # Ensure fakeroot is installed
```

Comment:
This isn't a change I made. It is in master, so I'm not sure why it's showing up here.